packages/core/src/main/conversation/ConversationService.ts

```diff
@@ -922,18 +922,15 @@ export class ConversationService {
     return (await request.response).buffer;
   }
 
-  public async addUser<T extends string | string[] | QualifiedId | QualifiedId[]>(
+  public async addUser(
     conversationId: string,
-    userIds: T,
-  ): Promise<T> {
+    userIds: string | string[] | QualifiedId | QualifiedId[],
```

Comment:
hmm this typing is getting a little concerning, but i guess its okay for this use case